app/src/main/java/com/cpr3663/cpr_scouting_app/PreMatch.java

```diff
@@ -100,10 +106,22 @@ protected void onCreate(Bundle savedInstanceState) {
             if (Globals.CurrentMatchNumber < Globals.MatchList.size()) {
```

Comment:
I feel like getMatchInfoRow() should check for a valid matchNumber.  We check for a Null Match right after this, so why do we need to check Globals.CurrentMatchNumber < Global.MatchList.size() ?  Seem it would be cleaner code to trust the class to protect itself.  Also better in case someone forgets to check somewhere else!

Can be fixed later.